### DISCUSSION TITLE
setTableRowClass() function removed?

### DISCUSSION THREAD

**Discussion by rootuser** [rappasoft/laravel-livewire-tables#1770] [Q&A] [answered]
Hi,

I would like to color the background of table rows depending on values of a certain field. 
In the documentation version 1 I found the **setTableRowClass()** function, see [https://rappasoft.com/docs/laravel-livewire-tables/v1/display/customizing-table-rows-and-cells#content-classes-ids-and-attributes](url), which seems capable of doing what I need.

But in version 3 (and also 2) there is no mentioning of  **setTableRowClass()**, nor do I find a replacement.

How can I implement the desired feature in the actual version?

Thank you!

Kind regards, 
Tom

**Comment by lrljoe** [comment, marked as answer]
It's called setTrAttributes now:
[https://rappasoft.com/docs/laravel-livewire-tables/v3/datatable/available-methods](https://rappasoft.com/docs/laravel-livewire-tables/v3/datatable/available-methods)
